### PR TITLE
Rename chem grenades w/ pen.

### PR DESCRIPTION
**Pull request by Iamgoofball**
requested by kor and others

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
you have to give [this](https://github.com/Iamgoofball/-tg-station/blob/baf5f83f70e0256ca0901ebc5d798057a2a48cf2/code/game/objects/items/weapons/grenades/chem_grenade.dm#L53) inheritance for it to work

File: code/game/objects/items/weapons/grenades/chem_grenade.dm [hunk 1]
```diff
@@ -50,6 +51,7 @@
 
 
 /obj/item/weapon/grenade/chem_grenade/attackby(obj/item/I, mob/user, params)
+	..()
```
Comment (ChangelingRain) [inline code/game/objects/items/weapons/grenades/chem_grenade.dm]: Proper inheritance; `return ..()` after everything else, and make sure everything here returns and doesn't get far enough to call ..()

Comment (Iamgoofball) [inline code/game/objects/items/weapons/grenades/chem_grenade.dm]: fake news

**Comment by TehZombehz** [comment]
So I can arm and lob several Thanks, Kevs at security.

**Comment by Tacolizard** [comment]
so can i rename my pyro grenades to heal grenades

**Comment by KorPhaeron** [comment]
stale

**Pull request closed without merging**